src/guide/views/menu/MenuExample.jsx

```diff
@@ -65,6 +69,7 @@ export default class MenuExample extends Component {
       <MenuItem
         key={index}
         label={item.name}
+        meta={item.serial}
         actions={[
           <Link key={0}>Allow</Link>,
           <Link key={1}>Block</Link>,
```

Comment:
Could you change the title of the example to "With meta and actions" or even create a separate example?